src/components/DayPickerNavigation.jsx

```diff
@@ -74,28 +89,33 @@ export default function DayPickerNavigation(props) {
   });
   const prevClassNames = cx('DayPickerNavigation__prev', {
     'DayPickerNavigation__prev--default': isDefaultNavPrev,
+    'DayPickerNavigation__prev--rtl': isRTL,
   });
   const nextClassNames = cx('DayPickerNavigation__next', {
     'DayPickerNavigation__next--default': isDefaultNavNext,
+    'DayPickerNavigation__next--rtl': isRTL,
   });
 
-  return (
-    <div className={navClassNames}>
 
-      {!isVerticalScrollable && (
-        <button
-          type="button"
-          aria-label={phrases.jumpToPrevMonth}
-          className={prevClassNames}
-          onClick={onPrevMonthClick}
-          onMouseUp={(e) => {
-            e.currentTarget.blur();
-          }}
-        >
-          {navPrevIcon}
-        </button>
-      )}
+  const prevButton = () => {
```

Comment:
i still don't think these should be functions; they should be conditionally rendered inline instead.